pkg/cli/cmds/log.go

```diff
@@ -73,10 +74,11 @@ func checkUnixTimestamp() error {
 }
 
 func setupLogging() {
-	flag.Set("v", strconv.Itoa(LogConfig.VLevel))
-	flag.Set("vmodule", LogConfig.VModule)
-	flag.Set("alsologtostderr", strconv.FormatBool(Debug))
-	flag.Set("logtostderr", strconv.FormatBool(!Debug))
+	fs := flag.NewFlagSet(os.Args[0], flag.ContinueOnError)
+	fs.Set("v", strconv.Itoa(LogConfig.VLevel))
+	fs.Set("vmodule", LogConfig.VModule)
+	fs.Set("alsologtostderr", strconv.FormatBool(Debug))
+	fs.Set("logtostderr", strconv.FormatBool(!Debug))
```

Comment:
I don't believe this will actually have the desired effect. The point was to set the values for the default global flagset `flag.CommandLine`, which is what's used when you call `flag.Set` instead of `flag.NewFlagSet().Set()`

Its possible we can just drop setting things here entirely, but I'd need to verify that by seeing if that removal breaks pass through of our debug, verbosity, and stderr logging flags. Probably we should delete this here and move to setting the flags individually in the args for the individual Kubernetes components.

Reply:
Might also reference the changes to rke2 from https://github.com/rancher/rke2/pull/3932

Given that upstream has dropped support for logtostderr/alsologtostderr/log-file/log-file-max-size, we should probably remove those. For the v/vmodule flags, we should pass those through to the individual Kubernetes component args instead of trying to set them here.

Reply:
okay so the latest update will basically move all the flag.Set to logging arguments, I have removed the logtostderr/alsologtostderr/log-file, except for kubeproxy where it seems it still uses it, other components will utilize the --v and --vmodule